products/bluebell/cli/src/main.rs

```diff
@@ -1,35 +1,16 @@
-// Importing necessary modules and types
-use bluebell::support::modules::BluebellModule;
-use bluebell::support::modules::ScillaDebugBuiltins;
-use std::ffi::CStr;
-
-use clap::{Parser, Subcommand, ValueEnum};
-// DEPRECATED
-// use inkwell::context::Context;
-// use inkwell::targets::{InitializationConfig, Target};
-use std::fs::File;
-use std::io::Read;
-use std::os::raw::c_char;
-use std::process;
-
 use bluebell::ast::nodes::NodeProgram;
-use bluebell::contract_executor::UnsafeContractExecutor;
-use bluebell::passes::debug_printer::DebugPrinter;
-
-use bluebell::support::evm::EvmCompiler;
-use bluebell::support::modules::{ScillaDefaultBuiltins, ScillaDefaultTypes};
-
-// use bluebell::llvm_ir_generator::LlvmIrGenerator;
-// use bluebell::support::llvm::{LlvmBackend, UnsafeLlvmTestExecutor};
-
-use bluebell::intermediate_representation::emitter::IrEmitter;
-use bluebell::intermediate_representation::pass_manager::PassManager;
-
 use bluebell::parser::lexer;
 use bluebell::parser::lexer::Lexer;
 use bluebell::parser::{parser, ParserError};
+use bluebell::support::evm::EvmCompiler;
+use bluebell::support::modules::ScillaDebugBuiltins;
```

Comment:
minor: you can get rust fmt to collapse all your imports like in this pr https://github.com/Zilliqa/zq2/pull/515

Reply:
Done